lib/linter.js

```diff
@@ -251,7 +249,7 @@ export default class Linter {
 
         if (code !== templateInfo.template) {
           let prefix;
-          if (templateInfo.templateMatch.type === 'template-tag') {
+          if (templateInfo.templateMatch && templateInfo.templateMatch.type === 'template-tag') {
```

Comment:
I do not think that it is possible for `templateInfo.templateMatch` to be undefined, I don't see why we need this initial guard